app/web/features/dashboard/Dashboard.tsx

```diff
@@ -43,27 +43,15 @@ export default function Dashboard() {
 
             <PageTitle>{t("dashboard:welcome")}</PageTitle>
 
-            <Alert severity="info" sx={{ marginBottom: theme.spacing(2) }}>
-              <Typography variant="body1">
-                <b>New Release Alert!</b> {dashboardNews["2025-05-26"]}
-              </Typography>
-            </Alert>
-
             <Alert severity="info" sx={{ marginBottom: theme.spacing(2) }}>
               <Typography variant="body1">
                 <b>New Blog Post!</b> Read it here:{" "}
```

Comment:
Should we make it something like `Exciting news! We just released v1, read more on the [blog](link)`?

Reply:
I was trying to make it more explicit as people not in tech might not understand what v1 means?